### PR TITLE
A playbook for releases

### PR DESCRIPTION
**Pull request by liztio**
Signed-off-by: liz <liz@heptio.com>

### CODE REVIEW

**Comment by liztio** [comment]
I have the draft of some documentation for certification, but I haven't been able to go through that process yet

File: docs/release.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Quickstart New Image Playbook
+
```
Comment (timothysc) [inline docs/release.md]: Add a purpose and scope section to outline what's in/out of this document.

File: docs/release.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+# Quickstart New Image Playbook
```
Comment (timothysc) [inline docs/release.md]: I struggle with the name here.  

Perhaps, Quickstart Update Guide.

File: docs/release.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+### 1. Build a new AMI using wardroom
+
+[Wardroom][wardroom] is a tool used to make Kubernetes-ready AMIs for use with
+the quickstart. The first step in a new quickstart image is building the image
+in Wardroom.
+
+* Checkout [`github.com/heptiolabs/wardroom`][wardroom].
+* You'll need Amazon creds in the environment. I recommend [aws-vault][vault].
```
Comment (timothysc) [inline docs/release.md]: s/I/we

File: docs/release.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  build_version=$(git rev-parse HEAD) -only ami-ubuntu-16.04 packer.json`.
+* Boot an instance with the image you created, just to verify that everything
+  worked. If you have the [`aws` cli][aws], you can run something like: `aws
+  ec2 run-instances --image-id <your ami ID> --instance-type t1.micro --region
+  us-east-1 --key-name <aws keypair> --security-groups <security group>`. Make
+  sure `<security group>` allows SSH to your node. `aws ec2 describe-instances
+  --instance-ids i-02cedcd163ae3a8ba | grep -i public` will get you the IP/DNS
+  name. Then you can SSH in with `ssh -i <your pem file> ubuntu@<public ip or
+  DNS>`.
+* Make sure the instance has the version of kubernetes you expect, and any other
```
Comment (timothysc) [inline docs/release.md]: What does "fix it" mean for a novice?

File: docs/release.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+### 2. Publish the AMIs. 
+
+Once you're happy with your AMI, it's time to copy the AMI to the other regions
+and make it public.
+
+* Again, Wardroom has [more detailed instructions][packer], but the basic
+  pattern is `python3 setup.py install` in the packer directory. Then `copy-ami
+  -i ami-78476f02 -r us-east-1`.
+* This process takes quite a while. Before too long, though, it's gonna spit out
```
Comment (timothysc) [inline docs/release.md]: s/spit out/emit.

File: docs/release.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+  some YAML. Move onto the next step with this YAML.
```
Comment (timothysc) [inline docs/release.md]: some YAML, which will be used in the next step.

File: docs/release.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+  create a new S3 bucket (`aws s3 mb`).
+* You can follow the boot on the command line, but you'll have an easier time of
+  it on the AWS web console. Log in, switch to the region you're using, and
+  navigate to cloudformation.
+* Once it's booted, look in the `Outputs` section for the outer formation (the
+  one without the random) characters at the end.
+* Find the `GetKubeConfigCommand` output, and copy it to a terminal. Sub in your
+  private key pair and run the command to get a kubeconig.
+* Validate your new config with `KUBECONFIG=./kubeconfig kubectl get nodes`.
+* Go to [scanner][scanner] and get a manieg
```
Comment (timothysc) [inline docs/release.md]: manieg?

File: docs/release.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+[scanner]: https://scanner.heptio.com/
+
+### 6. Send a PR to Amazon.
+
+The way the quickstart makes its way to Amazon is by PRing to Amazon's [upstream
+repo][amazon].
+
+* Open a PR from Heptio's repo to the [Amazon repo][amazon]. Important: PRs
+  should be sent to the `develop` branch, not the `master` branch.
+
```
Comment (timothysc) [inline docs/release.md]: Feel free to PR the conformance submission later.

File: docs/release.md (hunk 2 above)
Comment (liztio) [reply docs/release.md]: How about "Quickstart Release Guide?"

Comment (timothysc) [reply docs/release.md]: sgtm

File: docs/release.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+  navigate to cloudformation.
+* Once it's booted, look in the `Outputs` section for the outer formation (the
+  one without the random) characters at the end.
+* Find the `GetKubeConfigCommand` output, and copy it to a terminal. Sub in your
+  private key pair and run the command to get a kubeconig.
+* Validate your new config with `KUBECONFIG=./kubeconfig kubectl get nodes`.
+* Go to [scanner][scanner] and get a manieg
+* Apply it with your new config: `KUBECONFIG=./kubeconfig kubectl apply -f
+  https://scanner.heptio.com/...`
+* Go get some coffee. This'll be a while.
```
Comment (chuckha) [inline docs/release.md]: about 1 hour if that is useful information to have here

File: docs/release.md [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+* Once it's done, if it's a success, paste the results URL into your PR.
+* Terminate the cluster
+
+[testing]: https://github.com/heptio/aws-quickstart#testing-local-changes
+[scanner]: https://scanner.heptio.com/
+
+### 6. Send a PR to Amazon.
+
+The way the quickstart makes its way to Amazon is by PRing to Amazon's [upstream
```
Comment (chuckha) [inline docs/release.md]: [amazon] appears to be missing as a url

File: docs/release.md (hunk 4 above)
Comment (liztio) [reply docs/release.md]: I have no idea :( this is the point where a novice would have to dive deeper or ask one of us for help

**Pull request merged**